blueocean-dashboard/src/main/js/util/regex-weburl.js

```diff
@@ -0,0 +1,102 @@
+//
```

Comment:
Is this being used - can it be deleted? Or not used. I don't think it would do the right thing.